students/Timajeem/hw-1/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <style>
+       .textFirst{
+           color: red;
+       }
+    </style>
+    <title>hw_1</title>
+</head>
+<body>
+    <p class="textFirst">Lorem ipsum</p>
+    <p class="textSecond">Lorem ipsum</p>   
+    <script>
+        alert("String");
+    </script>
+    <script src="index.js"></script>
+</body>
+</html>
```

Comment:
Please add a new line at the bottom of your files.
Why? [link](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
How to? [link](https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files)